src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts

```diff
@@ -84,4 +88,76 @@ export class InputActionDialog {
   close() {
     this.modal.close();
   }
+
+  checkIfJenkinsIsUpAndProceedURL() {
+    let notification;
+    this.callJenkins()
+      .subscribe((response) => {
+        switch (response.status) {
+          case 200:
+            notification = {
+              message: `Got ${response.status}, Jenkins is up and running.`,
+              type: NotificationType.SUCCESS
+            };
+            this.notifications.message(notification as Notification);
+            this.invokeUrl(this.inputAction.proceedUrl);
+            return;
+          case 307:
+          case 302:
```

Comment:
I think the default behavior of the http library when hitting a 302 from `get(...)` is to follow the redirect automatically. I think we've observed this in practice here as well. So theoretically, it won't ever be 302. Up to you to keep this or not.